org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/folding/LSPFoldingReconcilingStrategy.java

```diff
@@ -167,8 +167,7 @@ private void applyFolding(List<FoldingRange> ranges) {
 			}
 			// send the calculated updates to the annotations to the
 			// annotation model
-			theProjectionAnnotationModel.modifyAnnotations(deletions.toArray(new Annotation[1]), additions,
-					modifications.toArray(new Annotation[0]));
+			theProjectionAnnotationModel.modifyAnnotations(deletions.toArray(new Annotation[1]), additions, new Annotation[0]);
```

Comment:
We still need to pass `modifications`  to `theProjectionAnnotationModel.modifyAnnotations`, as a subclass can have added some there, should we not?

Reply:
applyFolding is a private method. Subclasses can't have added something here. Or not?

Reply:
Ah, I think I got your point here now :-)